.travis.yml

```diff
@@ -1,35 +1,31 @@
 sudo: required
 
 services:
-- docker
+  - docker
 
 language: java
 jdk: openjdk8
 
-env:
-  global:
-    - SOURCE_IMAGE_NAME="collectionexercisesvc"
-    - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-collectionexercisesvc"
-
 before_install:
-- cp .maven.settings.xml $HOME/.m2/settings.xml
-- mvn fmt:check
+  - cp .maven.settings.xml $HOME/.m2/settings.xml
+  - mvn fmt:check
+  - docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
```

Comment:
docker login is called again below, I don't think it needs to be called here.

Reply:
needs to pull gcp image in integration tests

Reply:
👍 make sense, would be worth mentioning that change in the PR description.